website/docs/components/app-header/partials/code/how-to-use.md

```diff
@@ -0,0 +1,177 @@
+The other two slots are intended for consumer-provided controls. The `<:globalActions>` block should typically contain a “context switcher” (sometimes called an “org switcher” or “project switcher”). The `<:utilityActions>` block is used to provide utilities including “Help” and “User” menus and, optionally, “Search”.
+
+
+```handlebars
+<Hds::AppHeader>
+  <:logo>
+    <Doc::Placeholder @height="2em" @width="auto" @text="HomeLink" @background="#e4e4e4" />
+  </:logo>
+
+  <:globalActions>
+    <Doc::Placeholder @height="2em" @width="auto" @text="OrgSwitcher" @background="#e4e4e4" />
+  </:globalActions>
+
+  <:utilityActions>
+    <Doc::Placeholder @height="2em" @width="auto" @text="HelpMenu" @background="#e4e4e4" />
+    <Doc::Placeholder @height="2em" @width="auto" @text="UserMenu" @background="#e4e4e4" />
+  </:utilityActions>
+</Hds::AppHeader>
+```
+
+### Content
+
+#### HomeLink
+
+The `Hds::AppHeader::HomeLink` child component should be yielded within the `<:logo>` block. It provides consistent branding and navigates the user to the “home” or main dashboard page.
+
+It requires a value for the `@icon` and `@ariaLabel` arguments.
+
+It is built on top of the [`Hds::Interactive` component](/utilities/interactive), so it accepts all its routing arguments (eg. `@href`, `@route`, `@query`, `@model(s)`, etc.).
+
+Refer to the [Component API section](/components/app-header?tab=code#appheaderhomelink) for details.
+
+```handlebars
+<Hds::AppHeader>
+  <:logo>
+    <Hds::AppHeader::HomeLink 
+      @icon="hashicorp" 
+      @ariaLabel="HashiCorp home menu"
+      @href="/"
+    />
+  </:logo>
+  
+  <:globalActions>
+    <Doc::Placeholder @height="2em" @width="auto" @text="OrgSwitcher" @background="#e4e4e4" />
+  </:globalActions>
+  
+  <:utilityActions>
+    <Doc::Placeholder @height="2em" @width="auto" @text="HelpMenu" @background="#e4e4e4" />
+    <Doc::Placeholder @height="2em" @width="auto" @text="UserMenu" @background="#e4e4e4" />
+  </:utilityActions>
+</Hds::AppHeader>
+```
+
+The `HomeLink` also accepts optional arguments; for example, it’s possible to provide a custom color for the icon if needed:
+
+```handlebars
+<Hds::AppHeader>
+  <:logo>
+    <Hds::AppHeader::HomeLink 
+      @icon="terraform" 
+      @ariaLabel="Terraform home menu"
+      @color="var(--token-color-terraform-brand)"
+      @href="/"
+    />
+  </:logo>
+  
+  <:globalActions>
+    <Doc::Placeholder @height="2em" @width="auto" @text="OrgSwitcher" @background="#e4e4e4" />
+  </:globalActions>
+
+  <:utilityActions>
+    <Doc::Placeholder @height="2em" @width="auto" @text="HelpMenu" @background="#e4e4e4" />
+    <Doc::Placeholder @height="2em" @width="auto" @text="UserMenu" @background="#e4e4e4" />
+  </:utilityActions>
+</Hds::AppHeader>
+```
+
+#### Global actions
+
+Consumers should provide their own “context switcher” (e.g., “org switcher” or “project switcher”) control yielded within the 
+`<:globalActions>` block. HDS does not currently provide this component.
```

Comment:
```suggestion
Consumers should provide their own “context switcher” (e.g., “org switcher” or “project switcher”) control yielded within the `<:globalActions>` block. HDS does not currently provide this component.
```

Reply:
Also notice that in another part of the documentation we say consumers may use a `Dropdown` here. Maybe a contradiction to solve?